### DISCUSSION TITLE
[AKS] Network policy

### DISCUSSION THREAD

**Discussion by galibozek** [AzureCommunityPL/group#83] [Pomoc]
Jakich network policy użycie w kubernetesie, które odradzacie/polecacie?

**Comment by kaluzaaa** [comment]
- staw tyle klastrów, żeby nie było trzeba i wystarczył private link/nsg/azure firewall/appgw
- calico bardzo ładnie działa - ale w przypadku problemów ms nie wspiera, po za tą instalacją
- azure npm jest tylko produkcyjny z nazwy... wali się jak... https://github.com/Azure/AKS/issues/1810

**Reply by mgrabarz** [reply]
Dodatkowo, popraw mnie @kaluzaaa jeśli się mylę, preinstalowane Calico w AKS wspiera tylko Network Policy i nie obsługuje tych dodatkowych CRD (np Global Network Policy)...

**Reply by Gutek** [reply]
ono tam jest troche bardziej ograniczone chyba... miałem z tym duże problemy :( jak zwykle powiedziałem: zapiszę i opublikuje, jak zwykle nie zrobiłem :)

**Reply by galibozek** [reply]
czy może coś zmieniło się w temacie azure npm?

**Reply by kaluzaaa** [reply]
@galibozek nic :-)
